### DISCUSSION TITLE
Replace part of the URL with another string using custom bypasses

### DISCUSSION THREAD

**Discussion by aaronliu0130** [FastForwardTeam/FastForward#558] [Q&A] [answered]
How do I replace part of the URL with another string using custom bypasses? i.e. how do I define a rule set so `example.com/asdf` would automatically redirect to `zambo.com/asdf`?

**Comment by DumbAsFuck** [comment]
Well a late reply here, but if you haven't figured it out by now you could just run another tool to do that. Sorry I don't got anything better.

**Reply by aaronliu0130** [reply]
e.g.?

**Reply by DumbAsFuck** [reply]
Idk have you checked the extensions available for your browser? Pretty sure I saw something like that a while ago for some project I was working on. Wish I could remember the name.

**Comment by lostdusty** [comment, marked as answer]
I've done something similar to someone who wanted to replace the short imgbb domain, see here: https://github.com/FastForwardTeam/FastForward/issues/491#issuecomment-1129396612

Basically, the snippet is:
```js
domainBypass("example.com", () => {
        var a = window.location.href
        var finalurl = a.replace("example.com", "zambo.com")
        safelyNavigate(finalurl)
})
```

**Reply by lostdusty** [reply]
(Add that in custom bypass section)
